packages/discord-bot/src/handlers/activate.ts

```diff
@@ -24,21 +24,24 @@ export const activationHandler = async (
     await interaction.reply('Unable to create user account.');
     return;
   }
-  const baseURL = process.env.FRONTEND_URL;
-  if (!baseURL || baseURL === undefined) {
+  const server = process.env.SERVER;
+  if (!server || server === undefined) {
     await interaction.reply(
-      'ERROR: `FRONTEND_URL` not defined in environment variables. Contact praise admin'
+      'ERROR: `SERVER` not defined in environment variables. Contact praise admin'
     );
     return;
   }
+  const frontendUrl = `http://${
+    process.env.NODE_ENV === 'production' ? server : `${server}:3000`
+  }`; // Assume default craco port 3000 if not in production
```

Comment:
Same note as above. I personally think that server url & port should remain configurable for each service via the `env` files. 

We can also include a partially completed "default" `env` file for development vs production environments.

Reply:
Fixed